src/app/technical-record-search/technical-record-search.component.scss

```diff
@@ -1,4 +1,5 @@
 @import "node_modules/govuk-frontend/govuk/all";
```

Comment:
also i do not think you need this if you just `@import` from the root style of the app `app.component.scss`